src/common/lib/services/miniAppSdkHost.ts

```
@@ -0,0 +1,568 @@
+
+    // =========================================================================
+    // Cast composition
+    // =========================================================================
+    async composeCast(composeOptions: ComposeCastOptions): Promise<{ hash?: string } | null> {
+      // Open Warpcast compose with prefilled content
+      console.log('[MiniApp Host] composeCast:', composeOptions);
+      if (typeof window !== 'undefined') {
+        const params = new URLSearchParams();
+        if (composeOptions.text) params.set('text', composeOptions.text);
+        if (composeOptions.embeds?.length) {
+          composeOptions.embeds.forEach(embed => params.append('embeds[]', embed));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)